.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Security concerns
+    url: https://hackerone.com/github
+    about: Report security concerns on HackerOne.
+  - name: Questions about writing workflow or action files?
+    url: https://github.community/c/code-to-cloud/github-actions
```

Comment:
So much links are not related to this action IMO. I don't see at all someone thinking that those links may somehow useful :/